configuration/secrets.rst

```diff
@@ -0,0 +1,378 @@
+.. index::
+    single: Secrets
+
+How to Keep Sensitive Information Secret
+========================================
+
+.. versionadded:: 4.4
+
+    The Secrets management was introduced in Symfony 4.4.
+
+In :doc:`/configuration` and :doc:`/configuration/environment_variables`, you
+learned how to manage your application configuration. In this article you'll
+learn how to safely configure your application with sensitive information such
+as credentials, passwords, tokens, API keys without exposing them.
+
+.. note::
+
+    In order to use Symfony's secrets, you will need the `libsodium`_ PHP
```

Comment:
the extension is named just "sodium" when using PHP 7.2 (and is bundled into it, while you had to use pecl to install "libsodium" on 7.1)